src/plugins/plot/axis/YAxis.vue

```diff
@@ -63,67 +73,163 @@
 <script>
 import MctTicks from "../MctTicks.vue";
 import configStore from "../configuration/ConfigStore";
+import eventHelpers from "../lib/eventHelpers";
+
+const AXIS_PADDING = 20;
 
 export default {
     components: {
         MctTicks
     },
     inject: ['openmct', 'domainObject'],
     props: {
-        singleSeries: {
-            type: Boolean,
+        id: {
+            type: Number,
             default() {
-                return true;
+                return 1;
             }
         },
-        hasSameRangeValue: {
-            type: Boolean,
+        tickWidth: {
+            type: Number,
             default() {
-                return true;
+                return 0;
             }
         },
-        seriesModel: {
-            type: Object,
+        plotLeftTickWidth: {
+            type: Number,
             default() {
-                return {};
+                return 0;
             }
         },
-        tickWidth: {
-            type: Number,
+        multipleLeftAxes: {
+            type: Boolean,
             default() {
-                return 0;
+                return false;
+            }
+        },
+        position: {
+            type: String,
+            default() {
+                return 'left';
             }
         }
     },
     data() {
+        this.seriesModels = [];
```

Comment:
Please move to return block

Reply:
Doesn't that make it reactive? We don't need it to be.